diagnostic.md

```diff
@@ -7,40 +7,42 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+<!-- git clone https://github.com/jacobhereford/git-basics-diagnostic.git -->
```

Comment:
`git clone git@github.com/jacobhereford/git-basics-diagnostic.git`. You have the https url, we use the SSH.